Code/GraphMol/MolDraw2D/MolDraw2DHelpers.h

```diff
@@ -241,6 +243,8 @@ struct RDKIT_MOLDRAW2D_EXPORT MolDrawOptions {
   bool atomHighlightsAreCircles = false;  // forces atom highlights always to be
                                           // circles. Default (false) is to put
                                           // ellipses round longer labels.
+  MultiColourHighlightStyle multiColourHighlightStyle =
```

Comment:
Support for this also needs to be added to `updateMolDrawOptionsFromJSON()`

Reply:
I've had a stab at this, but I'm not convinced it's correct.  The test in testMolDraw2D.py passes, but maybe not for the right reason.

Reply:
> I've had a stab at this, but I'm not convinced it's correct. The test in testMolDraw2D.py passes, but maybe not for the right reason.

I think you forgot to commit the changed version of MolDraw2DUtils.cpp